rustler/src/export.rs

```diff
@@ -15,15 +15,27 @@
 /// The third argument is an `Option<fn(env: &Env, load_info: Term) -> bool>`. If this is
 /// `Some`, the function will execute when the NIF is first loaded by the BEAM.
 #[macro_export]
+#[deprecated(since = "0.22.0", note = "Please use `init!` instead.")]
 macro_rules! rustler_export_nifs {
+    ($name:expr, [$( $exported_nif:tt ),+,], $on_load:expr) => {
```

Comment:
I am curious, should'nt  the second branch  with `*`already cover this as well?

Reply:
Yeah, that seems to work. :+1: